src/engine/source/hlp/src/specificParsers.cpp

```diff
@@ -20,6 +21,32 @@
 #include <rapidjson/stringbuffer.h>
 #include <rapidjson/writer.h>
 
+// Complementary functions for JSON parser //
+enum class JSON_TYPE
+{
+    J_ANY,
+    J_STRING,
+    J_BOOL,
+    J_NUMBER,
+    J_OBJECT,
+    J_ARRAY,
+    J_NULL
+};
+
+static const std::unordered_map<std::string, JSON_TYPE> jsonTypes = {
+    {"string", JSON_TYPE::J_STRING},
+    {"bool", JSON_TYPE::J_BOOL},
+    {"number", JSON_TYPE::J_NUMBER},
+    {"object", JSON_TYPE::J_OBJECT},
+    {"array", JSON_TYPE::J_ARRAY},
+    {"null", JSON_TYPE::J_NULL},
+    {"any", JSON_TYPE::J_ANY}};
+
+bool validJsonType(const std::string& type)
```

Comment:
If the map is string_view, this prototype should change to string_view.

Reply:
ok